src/csi/pkg/pos/controllerserver.go

```diff
@@ -48,17 +49,25 @@ import (
 const (
 	CREATING string = "creating"
 	CREATED         = "created"
+	MAXIMUM_VOLUMES = 4
 )
 
 var errVolumeInCreation = status.Error(codes.Internal, "volume in creation")
 
 type controllerServer struct {
 	*csicommon.DefaultControllerServer
 	volumes map[string]*volume // volume id to volume struct
+	backends map[string]*backend //backend targetAddres to backend struct
 	mtx     sync.Mutex         // protect volume lock's map
 	mtx2    sync.Mutex         // for synchronizing requests to POS
 }
 
+type backend struct {
+    targetAddr string
+	provisionerIp string
```

Comment:
Regarding "provisionerIp",  using lowercase in params and uppercase in variables. I think this is confusing.
Is it possible to fixed it?

Reply:
This code was using "provisionerIP" and "provisionerIp" at the same time without difference, so i separate them. I use "provisionerIP" for whole addresses of POS  and "provisionerIp" for chosen POS address. I also think it should be modified in the not-confusing way. I am gonna change "provisionerIP" to the other name.